src/Cli/ConfigGenerator.cs

```diff
@@ -1094,7 +1094,7 @@ private static EntityGraphQLOptions ConstructUpdatedGraphQLDetails(Entity entity
         {
             //Updated GraphQL Type
             EntityGraphQLOptions graphQLType = (options.GraphQLType is not null) ? ConstructGraphQLTypeDetails(options.GraphQLType, null) : entity.GraphQL;
-            GraphQLOperation? graphQLOperation = null;
+            GraphQLOperation? graphQLOperation;
```

Comment:
do we not need to initialise the value anymore?

Reply:
No, Visual Studio complains of redundant assignment when initialized with `null`.